sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -603,7 +610,7 @@ def test_sliding_joint():
     assert C.masscenter.vel(P.frame) == v * P.frame.z
     assert C.ang_vel_in(P) == 0
     assert P.ang_vel_in(C) == 0
-
+test_sliding_joint()
```

Comment:
Has this been left in by error?

Reply:
Yes, that is my method of quickly testing a test. Will remove it.